test/unit/test/tmp_filename_test.cpp

```diff
@@ -0,0 +1,160 @@
+// -----------------------------------------------------------------------------------------------------
```

Comment:
Also here. Copy/Paste license from another file where the `-` are the correct length.